2023-12-20 00:34:16 ERROR Unable to initialize WebDriver
2023-12-20 00:34:16 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 394, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-CcJ6hTD8-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x56342fae0f83 <unknown>
#1 0x56342f799cf7 <unknown>
#2 0x56342f7e999e <unknown>
#3 0x56342f7e9aa1 <unknown>
#4 0x56342f834d64 <unknown>
#5 0x56342f8130dd <unknown>
#6 0x56342f832006 <unknown>
#7 0x56342f812e53 <unknown>
#8 0x56342f7dadd4 <unknown>
#9 0x56342f7dc1de <unknown>
#10 0x56342faa5531 <unknown>
#11 0x56342faa9455 <unknown>
#12 0x56342fa91f55 <unknown>
#13 0x56342faaa0ef <unknown>
#14 0x56342fa7599f <unknown>
#15 0x56342face008 <unknown>
#16 0x56342face1d7 <unknown>
#17 0x56342fae0124 <unknown>
#18 0x7ff057294ac3 <unknown>